cmake/cmaize/toolchain/toolchain.cmake

```diff
@@ -0,0 +1,192 @@
+include_guard()
+include(cmakepp_lang/cmakepp_lang)
+
+list(APPEND Toolchain_autopopulated_variable_names "CMAKE_C_COMPILER")
+list(APPEND Toolchain_autopopulated_variable_names "CMAKE_CXX_COMPILER")
```

Comment:
I recommend putting `_CMAIZE` at the beginning. At one point I had an explicit statement that any variable prefixed with that were reserved for CMaize.

Reply:
Should the variables also be in all caps?